packages/cache/__tests__/saveCache.test.ts

```diff
@@ -5,7 +5,7 @@ import * as cacheHttpClient from '../src/internal/cacheHttpClient'
 import * as cacheUtils from '../src/internal/cacheUtils'
 import {CacheFilename, CompressionMethod} from '../src/internal/constants'
 import * as tar from '../src/internal/tar'
-import {ITypedResponse} from '@actions/http-client/interfaces'
+import {TypedResponse} from '@actions/http-client/lib/interfaces'
```

Comment:
Having to put `/lib/` here is a Lerna thing now that we're using the same version of the package that's in the repo.